lib/widgets/text_form_custom.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+class TextFormCustom extends StatelessWidget {
+  TextFormCustom(this.labelText);
+
+  late final String labelText;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Text(
+          labelText.toUpperCase(),
+          style: TextStyle(
+            fontSize: 15.0,
+            fontWeight: FontWeight.w500,
+          ),
+        ),
+        SizedBox(
+          height: 2.0,
+        ),
+        TextFormField(
+          decoration: InputDecoration(
+            border: OutlineInputBorder(
+              borderRadius: BorderRadius.zero,
+            ),
+          ),
+        ),
```

Comment:
I'd use the Padding widget with the TextFormField as a child instead of a new widget to add space
Less widgets better performance.

Reply:
Im on it! But, simple question, Padding is not a Widget also like SizedBox?